tagstudio/src/core/utils/color.py

```diff
@@ -0,0 +1,15 @@
+from src.core.library.alchemy.models import Color, ColorNamespace
+from src.core.palette import ColorType
+
+def get_hex_color(color: Color, color_type: ColorType) -> str:
+    #TODO: everything
```

Comment:
this is obviously just a placeholder, I don't even know if this will be the final location for it, but for right now anything you want to do with the color function can be done here

Reply:
Assuming the `Color` object will take care of the color ID + namespace identification, I think this should take care of it!